packages/scene-composer/src/components/panels/scene-components/data-overlay/DataBindingMapEditor.spec.tsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import { act, render } from '@testing-library/react';
+import wrapper from '@awsui/components-react/test-utils/dom';
+
+import { mockProvider } from '../../../../../tests/components/panels/scene-components/MockComponents';
+import { IDataOverlayComponentInternal } from '../../../../store';
+import { Component } from '../../../../models/SceneModels';
+
+import { DataBindingMapEditor } from './DataBindingMapEditor';
+
+jest.mock('@awsui/components-react', () => ({
+  ...jest.requireActual('@awsui/components-react'),
+}));
+
+jest.mock('../../../../utils/mathUtils', () => ({
+  generateUUID: jest.fn(() => 'random-uuid'),
+}));
+
+let builderOnChangeCb;
```

Comment:
You don't need to do this. Instead in your mock code do:


```
return {
    ...originalModule,
    ValueDataBindingBuilder: jest.fn((props) => <div data-testid='ValueDataBindingBuilder'>{JSON.stringify(props)</div>);
    }
```

Then you just get it from the standard import:

```
import { ValueDataBindingBuilder as mockValueDataBindingBuilder } from '../ValueDataBindingBuilder'
```